public/pages/dashboard/ads.php

```diff
@@ -31,20 +35,18 @@
 
                 <div class="grid lg:grid-cols-3 md:grid-cols-2 grid-cols-1 gap-6 mt-6">
                     <?php
-                    /**
-                     * @var $ads
-                     */
                     foreach ($ads as $ad):?>
                         <div class="group rounded-xl bg-white dark:bg-slate-900 shadow hover:shadow-xl dark:hover:shadow-xl dark:shadow-gray-700 dark:hover:shadow-gray-700 overflow-hidden ease-in-out duration-500">
                             <div class="relative">
                                 <img src="<?= \App\Image::show($ad->image) ?>" alt="">
-
+                                <?php if ((new \App\Session())->getId() === $ad->user_id):?>
```

Comment:
What is that?

Reply:
Imtihon tugadiku ustoz boldi yigib oling .

Reply:
Abdurashid